mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/mapping/EmbeddableIdMappingTest.java

```diff
@@ -64,7 +62,7 @@ public void multiColumnEmbeddableIdMapping() {
 		OgmSession session = openSession();
 		Transaction transaction = session.beginTransaction();
 
-		assertDbObject(
+		assertDocument(
```

Comment:
An example of changes which should be avoided.

To be fair, the change itself isn't bad, it's just much easier for all if you do a method rename in a dedicated commit.